wps_tools/testing.py

```diff
@@ -16,7 +16,7 @@ def get(self, *args, **kwargs):
 
 
 def local_path(nc_file):
-    return f"file:///{resource_filename(__name__, 'data/' + nc_file)}"
+    return f"file:///{resource_filename('tests', 'data/' + nc_file)}"
```

Comment:
Did you find it more reliable to define `tests` instead?

Reply:
I had to change this since `thunderbird.common` is already in thunderbird's `tests` directory, whereas here its equivalent module is in a different `wps-tools` subdirectory.